.github/workflows/staging-build-tar.yml

```diff
@@ -459,3 +459,50 @@ jobs:
           working-directory: kibana/plugins/anomaly-detection-kibana-plugin
           command: yarn cy:run-with-security --config baseurl=http://localhost:5601
     
+  Test-Perf-Analyzer-No-Security:
+    runs-on: ubuntu-18.04
+    name: Test-Perf-Analyzer-No-Security
```

Comment:
This is not following the order we are doing in all the tests.
This test should be after "Test-AD-NoSec".

Also, the naming convention is not on par, you should use "Test-PA-NoSec".